lib/graphql/schema/member/has_fields.rb

```diff
@@ -53,8 +53,8 @@ def get_field(field_name)
         # @param field_defn [GraphQL::Schema::Field]
         # @return [void]
         def add_field(field_defn)
-          if CONFLICT_FIELD_NAMES.include?(field_defn.resolver_method)
-            warn "#{self.graphql_name}'s `field :#{field_defn.name}` conflicts with a built-in method, use `resolver_method:` to pick a different resolver method for this field (for example, `resolver_method: :resolve_#{field_defn.resolver_method}` and `def resolve_#{field_defn.resolver_method}`)"
+          if CONFLICT_FIELD_NAMES.include?(field_defn.original_name)
+            warn "#{self.graphql_name}'s `field :#{field_defn.name}` conflicts with a built-in method, use `resolver_method:` to pick a different resolver method for this field (for example, `resolver_method: :resolve_#{field_defn.name}` and `def resolve_#{field_defn.name}`)"
```

Comment:
I think all usages of `field_defn.name` here should be `field_defn.original_name`?

Otherwise `name` is usually camelized and would output `field :camelCase` and `:resolve_camelCase`

Reply:
If I am not mistaken `original_name` gets set to the name passed into the initialize method in `field.rb` which is the snake-case version. See below:

https://github.com/RobertWSaunders/graphql-ruby/blob/rework-conflicting-field-name-logic/lib%2Fgraphql%2Fschema%2Ffield.rb/#L141

Updating some tests for this now.

Reply:
`original_name` is more semantically correct imo, but since all the conflicted names are a single word only, it actually doesn't matter in reality.

Reply:
I agree, I've made it use `original_name`!